packages/okta-angular/README.md

```diff
@@ -144,6 +171,52 @@ const oktaConfig = {
 };
 ```
 
+## Reference
+### `oktaAuth`
+`oktaAuth` is the top-most component of `okta-angular`. This is where most of the configuration is provided.
```

Comment:
I'm finding this section a bit confusing because `oktaAuth` is a property that we add to the scope, after you've called `OktaAuthModule.initAuth(oktaConfig)` with the configuration options that your mentioning.  This isn't really clear, currently it looks like oktaAuth is a method I should call with the configuration options?

Reply:
+1. This is a bit confusing.